### ISSUE TITLE
[Question] Frontend IP

### ISSUE THREAD

**Issue by Moezenka** [citrix/citrix-k8s-ingress-controller#568]
**Describe the bug**
Hi,
We are currently annotating ingresses using the following annotation `ingress.citrix.com/frontend-ip: $FRONTEND_IP` in order to expose it on a different network segment.

When annotating the ingress it changes the IP, however when you delete the annotation, the ingress still retains the frontend-ip set earlier.

**To Reproduce**
```sh
# Create an ingress
$ kubectl create ingress ingress-demo --class=cic \
  --rule="ingress-demo.domain.com/*=demo:80"

# Verify that the ingress got created
$ kubectl get ing
NAME           CLASS   HOSTS                      ADDRESS         PORTS     AGE   LABELS
ingress-demo   cic     ingress-demo.domain.com   10.XX.XX.XX   80, 443   58s   <none>

# Annotate the ingress with the frontend annotation
$ kubectl annotate ingress ingress-demo ingress.citrix.com/frontend-ip=$FRONTEND_IP 
ingress.networking.k8s.io/ingress-demo annotated

# Verify that the ingress IP address changed
$ kubectl get ing
NAME           CLASS   HOSTS                      ADDRESS       PORTS     AGE
ingress-demo   cic     ingress-demo.domain.com   $FRONTEND_IP    80, 443   3m13s

# Delete the frontend annotation
$ kubectl annotate ingress ingress-demo ingress.citrix.com/frontend-ip-
ingress.networking.k8s.io/ingress-demo annotated

# Verify the IP adress of the ingress
$ kubectl get ing
NAME           CLASS   HOSTS                      ADDRESS       PORTS     AGE
ingress-demo   cic     ingress-demo.domain.com   $FRONTEND_IP   80, 443   7m12s
```

2. Version of the Citrix Ingress Controller
`image: quay.io/citrix/citrix-k8s-ingress-controller:1.26.7`

**Expected behavior**
I am not sure what the expected behavior is but I assume that it is supposed to revert the IP address set once you delete the annotation `ingress.citrix.com/frontend-ip $IP_ADDRESS`.

In short it should default to whatever IP was set prior to annotating the ingress with the frontend ip.

**Comment by sivakumarperumalla** [comment]
The observation is not related to kubernetes ingress resource ("citrix-k8s-ingress-controller" has no control on this). Can you please check with kubernetes team.

Thank you.

**Comment by ankits123** [comment]
@Moezenka thanks for pointing out.
We will discuss this and get back.
Usually when the frontendip( CS vs IP)  is modified,  it is equivalent to creating a new ingress and CS vserver.
This modify translates  to deleting the ingress and then modifying the ingress with new frontend ip annotation and then applying the ingress ...
